hackman/urls.py

```diff
@@ -56,8 +56,8 @@ def robots(request: HttpRequest) -> HttpResponse:
         auth_views.PasswordResetDoneView.as_view(),
         name="password_reset_done",
     ),
-    re_path(
-        "reset/(?P<uidb64>[0-9A-Za-z_-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/",  # noqa
```

Comment:
This looks like some sort of input validation.  Is it safe/wise to remove this?  Is it only flake8 that gets upset about this?